crates/blockifier/src/execution/contract_class.rs

```diff
@@ -86,7 +88,7 @@ impl ContractClassV0 {
             + self.n_builtins()
             + self.bytecode_length()
             + 1; // Hinted class hash.
-        // The hashed data size is approximately the number of hashes (invoked in hash chains).
+                 // The hashed data size is approximately the number of hashes (invoked in hash chains).
```

Comment:
added white spaces?

Reply:
The whitespaces exist because of `cargo fmt`. This change wasn't done manually.